src/image/manifest.rs

```diff
@@ -253,10 +253,9 @@ mod tests {
             .media_type(MediaType::ImageConfig)
             .size(7023u64)
             .digest(
-                Sha256Digest::from_str(
-                    "b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7",
-                )
-                .unwrap(),
+                "b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7"
+                    .parse::<Sha256Digest>()
+                    .unwrap(),
```

Comment:
nit: We could also return a `Result<ImageManifest>` here and use the `?` operator instead of unwrapping.

Reply:
This is just in testing code, and the rest of this function uses unwraps for similar "should not fail" situations (using the builder API).

I think there's an interesting sub-thread here in that ideally we'd support constructing a `Sha256Digest` from constant input at compile time, but AFAIK [const eval](https://doc.rust-lang.org/reference/const_eval.html) doesn't yet support unwrapping.